### ISSUE TITLE
Lint/UnneededDisable cop can't be disabled inline

### ISSUE THREAD

**Issue by jfelchner** [bbatsov/rubocop#1959]
I tried to disable the `Lint/UnneededDisable` on a line that has some standard disablings that I wrap all my validations in and it threw an error that `Lint/UnneededDisable` didn't need to be disabled :grinning:

**Comment by rrosenblum** [comment]
It doesn't seem to be able to be disabled at all. My `.rubocop.yml` file contains
```yml
Lint/UnneededDisable:
  Enabled: false
```
and RuboCop will still report offenses for `Lint/UnneededDisable`.

**Comment by rrosenblum** [comment]
Should this cop be allowed to be disabled?

**Comment by jfelchner** [comment]
Of course. Why wouldn't it be able to be?
